components/molecules/FeatureBlock.js

```diff
@@ -12,12 +12,8 @@ export default function FeatureBlock(props) {
         id="feature-block"
         data-testid="featureBlock"
       >
-        {/* <div className="bg-splash-page bg-cover bg-center w-full py-32  ">
-  
-      </div> */}
         <div className="aspect-w-5 aspect-h-2 sm:aspect-h-1 xl:ml-3 w-full min-w-max mr-break-out">
           <Image
-            // src="/autumn-leaves-woman-hands.png"
             src={
               'https://www.canada.ca/content/dam/decd-endc/images/autumn_leaves_woman_hands.png'
```

Comment:
the image should be defined in AEM not hardcoded, maybe just leave as it was until this gets decided

Reply:
Ok I'll re-add the note and also add that this should be acquired from AEM.